deeprank/config/chemicals.py

```diff
@@ -70,4 +70,17 @@
     'THR': 'polar',
     'TRP': 'polar',
     'TYR': 'polar'
-    }
\ No newline at end of file
+    }
+
+
+# atom vdw radius
+# https://en.wikipedia.org/wiki/Van_der_Waals_radius
```

Comment:
Could we use something else than Wikipedia ?
The python package Mendeleev has vdw radius for all atoms I think

Reply:
We need vdw_radius for only a few elements, maybe too heavy to import a package to do that.
Changed the reference to a book.